### DISCUSSION TITLE
This entire category could do with a cleanup...

### DISCUSSION THREAD

**Discussion by patrickhlauke** [community/community#49127] [Accessibility] [answered]
### Select Topic Area

General

### Body

Skimming over the accessibility category here, the vast majority of discussion topics have absolutely nothing to do with accessibility. Is this monitored and maintained by anybody? Would be good to move/relabel all the things here that are not #a11y related...

**Comment by ettaboyle** [comment, marked as answer]
Thank you for the feedback @patrickhlauke 👋🏻 Yes, there is a dedicated team of community managers who are actively working to keep the discussions space healthy. We're working on it - and glad you're here. Thanks!

**Comment by cehfisher** [comment]
Thanks for the feedback @patrickhlauke! We are actively working on the clean-up process and ways to keep it up-to-date going forward.
